fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilderForCreateView.java

```diff
@@ -56,6 +56,10 @@
 
 /**LogicalPlanBuilderForCreateView*/
 public class LogicalPlanBuilderForCreateView extends LogicalPlanBuilder {
+    public LogicalPlanBuilderForCreateView() {
+        super(null);
```

Comment:
i think it should use hint too? for example we could use hint to change sql mode, if create view cannot process it correctly it could generate wrong column type. or cannot analyze create view statement correctly.

Reply:
done